kinesis-s3/stop.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# Source library 
+. ../utils/helper.sh
+
+source config/demo.cfg
+SCHEMA_REGISTRY_CONFIG_FILE=$HOME/.ccloud/config
```

Comment:
This script is defaulting to cloud SR, while `start.sh` defaults to local SR

Reply:
That was intentional as I'm toggling between the two at the moment (we can discuss why offline).

Long term: it will be CCSR